presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -505,23 +507,33 @@ public MaterializedResult execute(String sql)
     public MaterializedResult execute(Session session, String sql)
     {
         try {
-            return executeWithRetryStrategies(session, sql, ImmutableList.of());
+            List<ExecutionStrategy> executionStrategies = getExecutionStrategies(session);
```

Comment:
Can't we just inline this? I don't see `getExecutionStrategies(Session session)` being called from any other place.